packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -177,17 +177,13 @@ def get_configuration_parameters_with_type_and_default_values(
         default_value = metadata.default_value
         parameter_type = _get_python_type_as_str(metadata.type, metadata.repeated)
         if isinstance(default_value, str):
-            default_value = f'"{default_value}"'
+            default_value = default_value.encode("unicode-escape")
```

Comment:
This generates `bytes` literals, not `str` literals. Also, the way it does this is somewhat convoluted:
- `default_value.encode("unicode-escape")` evaluates to a `bytes` containing the escaped string, without any surrounding quotes.
- `f"... = {default_value}"` prints a `bytes` literal or b-string, which escapes everything again. I'm not sure if this generates the right number of quotes and backslashes because none of the test cases use these characters.

Can you use `default_value = repr(default_value)` instead?

Reply:
Sure. I have updated with `repr`.